WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -624,7 +621,8 @@ function(el) {
 		var parentEl = document.getElementById(this._htmlElId);
 		parentEl.appendChild(this._newTaskInputEl);
 
-		this._newTaskInputEl.style.padding  = "0px";
+		this._newTaskInputEl.style.padding      = "0px";
+		this._newTaskInputEl.style.paddingLeft  = "14px";
```

Comment:
oh ok. Thanks. 
Any specific reason why we increased the specificity for the selector? Just curious to know about that.